### ISSUE TITLE
Back button on search form does not reselect place

### ISSUE THREAD

**Issue by benwbrum** [FreeUKGen/FreeCENMigration#101]
To investigate and maybe not fix.

From the testing, it would be nice if we could make the browser back button work like the 'revise search' button does.  I have no earthly idea how to implement this, however:

The only "problem" as such I am getting is when using my web browser back button to review the search criteria 

the "Place' is slipping. Getting different issues.

I confirm these issues do not occur when using the built in navigations tools.

Round 1 - Entered a name, Selected Aberdeenshire but did not select a place and year 1861. Observed results 

list without issue and clicked on the browser back button.

Result was as below - you will see that I have still got the search criteria but have now got Cornwall place names 

(This was what I searched previous to this search).

**Comment by benwbrum** [comment]
More comments from Beki:
Several attempts of searches made using various criteria and using the back button after result obtained similar results.

I understand that the design is preferring the use of the Revise Search button but many users naturally will still use the back button. 

As the back button mostly produces the correct return result it would be great if we could get the 'Place' issue sorted.

My only suggestion for an addition would be a refresh/reset button for the form. But it may not be necessary.

I hope this is of some sort of assistance.

**Comment by israelriibeiro** [comment]
Hi, @benwbrum.

I create a branch here with a peace of code that maybe will resolve this problem. But, first, let me explain what i did: 

I saw that, when click on web browser back button,  rails won't log anything, but when refresh the page, it show everything on your terminal again. Said that: i tried not cache on the web browser. After that, i notice that params[:search_id] was "empty" when i clicked back button. So i tried to check if session, rails session, was receive anything from query. And i did. 

I found the problem, but it can't, i think, the final solution. This maybe will bring problem in another parts of the application, we should check this, before i do the pull request.

[Issue 101](https://github.com/FreeUKGen/MyopicVicar/commit/92dff07344b19a6bfa109a55a838b4b59e1a8ef1)
